src/app/dynamic/enterprise/quotas/project-quota-dialog/component.ts

```diff
@@ -66,17 +67,18 @@ export class ProjectQuotaDialogComponent implements OnInit, OnDestroy {
   }
 
   ngOnInit(): void {
-    this._initForm();
-    this._getProjects();
+    this._setSelectedQuota();
     this._getQuotas();
```

Comment:
why the order has changed?

Reply:
Getting it aligned with the order of methods below